sunpy/map/mapsequence.py

```diff
@@ -362,18 +361,15 @@ def updatefig(i, im, annotate, ani_data, removes):
 
             im.set_array(ani_data[i].data)
             im.set_cmap(kwargs.get('cmap', ani_data[i].plot_settings['cmap']))
-            norm = deepcopy(kwargs.get('norm', ani_data[i].plot_settings['norm']))
+            norm = kwargs.get('norm') or ani_data[i].plot_settings["norm"] if "norm" in ani_data[i].plot_settings else None
```

Comment:
The `deepcopy` here is very intentional because if the `norm` does not have limits already set, they will be set when applied to the image and thus the one passed in will be modified in place.

Reply:
Added it back.